example/ios/Runner.xcworkspace/xcshareddata/WorkspaceSettings.xcsettings

```diff
@@ -2,7 +2,7 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
-	<key>BuildSystemType</key>
```

Comment:
Is it necessary to check in these preferences?

Reply:
We can revert this to original.